src/server/qgsserverapiutils.cpp

```diff
@@ -0,0 +1,162 @@
+/***************************************************************************
+                          qgsserverapiutils.cpp
+
+  Class defining utilities for QGIS server APIs.
+  -------------------
+  begin                : 2019-04-16
+  copyright            : (C) 2019 by Alessandro Pasotti
+  email                : elpaso at itopen dot it
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "qgsserverapiutils.h"
+#include "qgsrectangle.h"
+#include "qgsvectorlayer.h"
+#include "qgscoordinatereferencesystem.h"
+#include "qgsserverprojectutils.h"
+#include "qgsmessagelog.h"
+
+#include "nlohmann/json.hpp"
+
+#include <QUrl>
+
+QgsRectangle QgsServerApiUtils::parseBbox( const QString &bbox )
```

Comment:
Do you think that `QgsServerParameterDefinition::toRectangle()` method could be reused in some way?

Reply:
I'm afraid not, but looking again to my implementation I see that I made a mistake in case we get 6 parameters with Z. I'll fix it.